crates/holochain/src/conductor/conductor.rs

```diff
@@ -1359,8 +1362,13 @@ mod clone_cell_impls {
                 )
                 .await?;
 
+            let clone_cell_id = if let CellInfo::Cloned(clone_cell) = &installed_clone_cell {
+                clone_cell.cell_id.clone()
+            } else {
+                panic!("create clone cell: created cell is not of type 'cloned'");
+            };
```

Comment:
This panic could be removed (and I think it should) by making `add_clone_cell_to_app` just return `Cell`

Reply:
I had considered that too but didn't go for it. Okay